ccw.core/src/java/ccw/core/StaticStrings.java

```diff
@@ -0,0 +1,9 @@
+package ccw.core;
+
+public class StaticStrings {
```

Comment:
I called it that way because it includes (or better, it will include, in #43) other strings necessary to the program, we could divide in SystemPropertiesConstants, ColorConstants, ... have a look at (this)[https://github.com/arichiardi/ccw/blob/master/ccw.core/src/java/ccw/core/StaticStrings.java#L1]